src/content/pages/index.html

```diff
@@ -66,59 +66,62 @@ <h1 class="section-title">{{ site.homepage.title }}</h1>
-                        | replace("DATE", version_date)
-                    %}
-                    <svg width="16" height="8" viewBox="0 0 16 8" class="c-icon" role="img" aria-label="npm">
-                        <path d="M0 0.888885H16V6.22222H8V7.11111H4.44444V6.22222H0V0.888885ZM0.888889 5.33333H2.66667V2.66666H3.55556V5.33333H4.44444V1.77777H0.888889V5.33333ZM5.33333 1.77777V6.22222H7.11111V5.33333H8.88889V1.77777H5.33333ZM7.11111 2.66666H8V4.44444H7.11111V2.66666ZM9.77778 1.77777V5.33333H11.5556V2.66666H12.4444V5.33333H13.3333V2.66666H14.2222V5.33333H15.1111V1.77777H9.77778Z" fill="currentColor" />
-                    </svg>
-                    <span>{{ dateline }}</span>
-                </dd>
-                {%- endif %}
-                <dt>{{ site.homepage.versions.upcoming }}</dt>
-                <dd>
-                    {% set version_date = stats.nextVersionDate | shortDateFromISO %}
-                    {% set dateline = site.homepage.versions.dateline | safe
-                        | replace("VERSION", "<a class=\"text-dark\">" + stats.nextVersion + "</a>")
-                        | replace("DATE", version_date)
-                    %}
-                    <svg width="16" height="16" viewBox="0 0 16 16" class="c-icon" role="img" aria-label="Github">
-                        <path d="M8 0.200012C3.6 0.200012 0 3.80001 0 8.20001C0 11.7333 2.26667 14.7333 5.46667 15.8C5.86667 15.8667 6 15.6 6 15.4C6 15.2 6 14.7333 6 14.0667C3.8 14.5333 3.33333 13 3.33333 13C3 12.0667 2.46667 11.8 2.46667 11.8C1.66667 11.3333 2.46667 11.3333 2.46667 11.3333C3.26667 11.4 3.66667 12.1333 3.66667 12.1333C4.4 13.3333 5.53333 13 6 12.8C6.06667 12.2667 6.26667 11.9333 6.53333 11.7333C4.73333 11.5333 2.86667 10.8667 2.86667 7.80001C2.86667 6.93335 3.2 6.20001 3.66667 5.66668C3.66667 5.40001 3.33333 4.60001 3.8 3.53335C3.8 3.53335 4.46667 3.33335 6 4.33335C6.66667 4.13335 7.33333 4.06668 8 4.06668C8.66667 4.06668 9.33333 4.13335 10 4.33335C11.5333 3.26668 12.2 3.53335 12.2 3.53335C12.6667 4.66668 12.3333 5.46668 12.2667 5.66668C12.8 6.20001 13.0667 6.93335 13.0667 7.80001C13.0667 10.8667 11.2 11.5333 9.4 11.7333C9.66667 12 9.93333 12.4667 9.93333 13.2C9.93333 14.2667 9.93333 15.1333 9.93333 15.4C9.93333 15.6 10.0667 15.8667 10.4667 15.8C13.6667 14.7333 15.9333 11.7333 15.9333 8.20001C16 3.80001 12.4 0.200012 8 0.200012Z" fill="currentColor" />
-                    </svg>
-                    <span>{{ dateline }}</span>
-                </dd>
-                {% if not stats.currentVersionIsPrerelease -%}
-                <dt>{{ site.homepage.versions.development }}</dt>
-                <dd>
-                    {% set version_date = stats.lastCommitDate | shortDateFromISO %}
-                    {% set dateline = site.homepage.versions.dateline | safe | replace("VERSION", "<a class=\"text-dark\">HEAD</a>") | replace("DATE", version_date) %}
+            <div class="eslint-versions-container">
```

Comment:
Two small notes about the layout after this change, I'm not sure how important they are and whether they're intentional:

1. The ad and the versions are moved up a bit (when compared to the current eslint.org homepage).

![image](https://github.com/eslint/eslint.org/assets/44349756/0d439903-9032-4c79-a7a4-88c9fa472c69)

2. On a mobile screen, the Version Support link seems too close to the horizontal line below it.

![image](https://github.com/eslint/eslint.org/assets/44349756/bed94b6a-d42e-46fd-962a-074fb67f8e8e)

Reply:
For 1, best I can tell, adding the "Version Support" link pushed everything up, but I have no idea why. Someone's who done web development more recently than I can probably figure it out pretty quickly. :smile:

For 2, I've updated the styles.

Reply:
1. Everything moved up a little bit because of 
`	padding-block-end: 2rem;
	padding-bottom: 2rem;`

This padding seems to force content to move upwards

Reply:
In 1, the ad and version shift slightly upward due to the `align-items: center` property applied to both the .hero>.content-container and .hero .grid elements. The `align-items: center` property attempts to center-align the child elements, so adding a new element to the right column causes it to shift up slightly. 


![Screenshot 2024-07-10 at 11 52 20 PM](https://github.com/eslint/eslint.org/assets/30730124/65c2cd31-7a6e-48f7-93bb-84075fee8947)

To fix this, we can do the below steps:

step 1:  align `.hero>.content-container` element normally 
![image](https://github.com/eslint/eslint.org/assets/30730124/6c54a8f1-70de-4b4e-af92-cbb4d0b3e12c)



step 2: The left grid `span-1-7 content-container` element has some padding applied. Apply the same padding to the right grid as well.

![image](https://github.com/eslint/eslint.org/assets/30730124/67c03885-f1f6-4f2d-9323-4f3392d15fac)


![image](https://github.com/eslint/eslint.org/assets/30730124/897fd8a4-9b9f-4ea6-a865-bfb0cabb0559)

Reply:
Thanks!